### PR TITLE
Added note about checking for merged branches

### PR DESCRIPTION
**Pull request by RichardLitt**
This should check if your commits are fully merged or not. Mostly taken from
http://stackoverflow.com/questions/7548926/git-and-the-branch-x-is-not-fully-merged-error.

### CODE REVIEW

**Comment by RichardLitt** [comment]
@k88hudson Care to double check?

File: README.md [hunk 1]
```diff
@@ -650,6 +651,21 @@ If someone has sent you a pull request on GitHub, but then deleted their origina
 
 After commiting, change the author of the previous commit. To do this, see how to [change author](#commit-wrong-author). Then, apply whatever changes needed on to, and make a new pull request.
 
+<a name="merged-commits"></a>
```
Comment (k88hudson) [inline README.md]: Looks like this doesn't match up wit the link name

**Comment by k88hudson** [comment]
Nice!

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: Fixed!

**Comment by RichardLitt** [comment]
Thanks!

**Pull request merged**